chamber/utils/json.py

```diff
@@ -0,0 +1,16 @@
+from django.core.serializers.json import DjangoJSONEncoder
+
+
+class ChamberJSONEncoder(DjangoJSONEncoder):
+
+    def default(self, val):
+        if isinstance(val, bytes):
+            return {
+                '__type__': bytes.__name__,
+                '__value__': [x for x in val],
```

Comment:
Why not `x.decode()`?

Reply:
We can't generally really be sure about the encoding used. On the other hand I think we can be sure that `bytes` is always iterable so no test is neccessary.

Reply:
OK, understood. But I disagree with the missing test, let @matllubos decide.

Reply:
and yes you are true, the encoder should be tested